scripts/bump.sh

```diff
@@ -9,50 +9,63 @@ DEPENDENCIES=(
 # Variables
 NB_ARGS="$#"
 
+# ------------------------------- Tool functions -------------------------------
+
+log_any () {
+	echo -e "[\e[$1m$2\e[0m] $3"
+}
+
+log_error   () { log_any "31" "!" $1 }
```

Comment:
```suggestion
log_error   () { log_any "31" "!" "$1"; }
```
else it doesn't work on my machine :confused: 
the `;` solves a `./scripts/bump.sh: line 159: syntax error: unexpected end of file` error
the `"` allows to pass multiple words string as a single argument

to repeat on each line :wink:

Reply:
you are absolutely right, and that entirely my fault for not testing the new script in `DRY_RUN` mode :face_exhaling: 
i change that :+1:

Reply:
`;` should be addressed in f9c6a87
`"` should be addressed in 91d2c08

Reply:
perfect!